sensor_msgs/Image rgb_image

sensor_msgs/Image depth_image

sensor_msgs/CameraInfo camera_info

---

vision_msgs/Detection3DArray detections

vision_msgs/BoundingBox2D[] bboxes

sensor_msgs/Image label_image_raw

sensor_msgs/Image label_image_color